utils/codegen/docker/rgen.Dockerfile

```diff
@@ -0,0 +1,54 @@
+# docker build -t ghcr.io/ika-rwth-aachen/etsi_its_messages:rgen -f rgen.Dockerfile .
```

Comment:
We should poush this image to the repo's container registry.

We can leave this thread open until all other are resolved and we are ready to merge.